custom_typings/bower-config.d.ts

```diff
@@ -0,0 +1,23 @@
+// https://github.com/bower/spec/blob/master/json.md
+interface BowerConfig {
```

Comment:
If these are just global, and not typing a particular module, I'm not sure they should be in custom_typings. They can just be a .ts file in src/

Reply:
Turned them into modules in `src/`.